dbt/include/firebolt/macros/adapters.sql

```diff
@@ -183,90 +183,6 @@
   {{ return(info_table) }}
 {% endmacro %}
 
-
-{% macro firebolt__create_table_as(temporary,
```

Comment:
Why don't we need these anymore? Can we use the default dbt implementation now?

Reply:
They've a part of the big dbt refactor. Now this (as well as the view macros) live in their respective subdirectories. e.g. dbt/include/firebolt/macros/relations/table/create.sql instead of everything living in the `adapters.sql`